src/styles/blocks/button.scss

```diff
@@ -0,0 +1,45 @@
+.button {
+  display: block;
+  width: 166px;
+  height: 40px;
+  margin-left: 17px;
+  margin-top: 16px;
+  border: none;
+  border-radius: 5px;
+  background-color: #00acdc;
+
+  &:hover {
+    display: block;
+    width: 166px;
+    height: 40px;
+    margin-left: 17px;
+    margin-top: 16px;
+    border: 1px solid #00acdc;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should not add a border in the hover state that wasn't there before as it can cause the button to shift due to the change in its box model. If you want to keep the button the same size, consider using an invisible border in the non-hover state, or adjust padding/margin to compensate for the added border.